apps/developer-hub/src/components/Error/index.module.scss

```diff
@@ -0,0 +1,63 @@
+@use "@pythnetwork/component-library/theme";
```

Comment:
There's an [ErrorPage component in the component library](https://github.com/pyth-network/pyth-crosschain/tree/main/packages/component-library/src/ErrorPage) now which I recently extracted out of insights hub; I'd suggest using that instead of having a bespoke one for this app (e.g. see how I'm doing it in [the Entropy explorer app](https://github.com/pyth-network/pyth-crosschain/blob/main/apps/entropy-explorer/src/app/error.ts)